test/cypress/integration/group-member-removal.spec.js

```diff
@@ -82,8 +82,8 @@ describe('Group - Removing a member', () => {
   })
 
   it('user2 rejoins groupA', () => {
-    // Wait for contracts to sync before visiting invitationLink. TODO avoid .wait().
-    cy.wait(500); // eslint-disable-line
+    // TODO: Avoid .wait() call. It waits for contracts to sync before visiting invitationLink.
+    cy.wait(2000); // eslint-disable-line
```

Comment:
Increasing this to 2000 to help maybe with Travis isn't ideal, maybe we can use this PR to fix this TODO once and for all?

The `giLogin` command for example should wait correctly, as it has:

```js
  cy.getByDT('app').then(([el]) => {
    cy.get(el).should('have.attr', 'data-logged-in', 'yes')
    cy.get(el).should('have.attr', 'data-sync', '')
  })
```

Which waits for us to be logged in and all contracts to finish syncing.

If at this point in the test we're not logged in and you still want to wait for all contracts to finish syncing, I think you could do:

```js
  cy.getByDT('app').then(([el]) => {
    cy.get(el).should('have.attr', 'data-sync', '')
  })
```

But double-check me on that.

Reply:
Hi @taoeffect . Can we create a new branch which is for removing `cy.wait` in cypress test?
This `cy.wait` is not only related to the chatroom basic functions and futher more in the future.
So I wanna create a new branch and work inside.

Reply:
We sync chatroom contract after syncing group contract is finished.
So there is a time when `data-sync` is `""` just after group contract is synced and just before chatroom contract starts to be synced.

Reply:
Leave this right now. This seems like the error from the core chelonia.
So we are going to fix this alter.